ports/espressif/mpconfigport.mk

```diff
@@ -36,6 +36,7 @@ CIRCUITPY_I2CTARGET ?= 1
 CIRCUITPY_IMAGECAPTURE = 0
 CIRCUITPY_MEMORYMAP ?= 1
 CIRCUITPY_NVM ?= 1
+CIRCUITPY_PARALLELDISPLAY ?= 0 # Turn off because it uses the old I2S driver which conflicts with the new ADC driver.
```

Comment:
Is this off temporarily until it's rewritten? Open a reminder issue?

Reply:
Just a reminder that due to how make processes things, a same-line comment like this breaks checking configuration options for 1/0: The value of CIRCUITPY_PARALLELDISPLAY becomes "0 " with a trailing space, and so comparing it to the string "0" results in "not equal". Place the comment on the line above if it needs to stay.